mix.exs

```diff
@@ -36,10 +36,29 @@ defmodule Membrane.Template.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.7.0"},
-      {:ex_doc, "~> 0.24", only: :dev, runtime: false},
+      {:membrane_core, "~> 0.10.0"},
+      {:bimap, "~> 1.2"},
```

Comment:
Sorry, this one bothers me. It interrupts a nice wall of membrane plugins. Group it with qex in their own group 😅 Especially that qex's positioning seems somewhat random as well